docs/tutorials/clickstream-docker.rst

```diff
@@ -26,13 +26,13 @@ your local host.
 
 .. code:: bash
 
-    $ docker run -p 33000:3000 -it confluentinc/ksql-clickstream-demo:0.5 bash
+    $ docker run -p 33000:3000 -it confluentinc/ksql-clickstream-demo:4.1 bash
 
 Your output should resemble:
 
 .. code:: bash
 
-    Unable to find image 'confluentinc/ksql-clickstream-demo:0.5' locally
+    Unable to find image 'confluentinc/ksql-clickstream-demo:4.1' locally
```

Comment:
are we sure we are publishing images with these versions? Unless we are manually doing it, I doubt dockerhub will be updated with a tag with that name. It doesn't exist right now: https://hub.docker.com/r/confluentinc/ksql-clickstream-demo/tags/

Best to leave the docker tags out of the renames.

Reply:
Won't these be released as part of 4.1?

Reply:
They will not be released automatically since there is no automatic pipeline to build these images. And when they are released they will be versioned like `4.1.0`, etc. 

So either way the above is incorrect. I would leave it out now. After the release is cut we can upload the docker images and then update the docs accordingly.